design/tracker_queries.go

```diff
@@ -39,6 +39,7 @@ var trackerQueryAttributes = a.Type("TrackerQueryAttributes", func() {
 var trackerQueryRelationships = a.Type("TrackerQueryRelations", func() {
 	a.Attribute("tracker", relationKindUUID, "This defines the related tracker")
 	a.Attribute("space", relationSpaces, "This defines the owning space")
+	a.Attribute("baseType", relationBaseType, "This defines type of Work Item")
```

Comment:
Please call this `workItemType` instead and give a more meaningful description such as `"Defines what work item type to use when instantiating work items using this tracker query."`

Reply:
done here 
4e28161